hs_tracking/management/commands/stats.py

```diff
@@ -231,6 +270,70 @@ def yesterdays_variables(self, lookback=1):
                       vals]
             print('|'.join(values))
 
+    def publications_between_dates(self, start_date, end_date, options):
+        '''
+        Print publication stats between start and end dates
+        '''
+
+        def print_resource_info_from_dates(dates):
+            for d in dates:
+                print("-" * 80)
+                try:
+                    print(f"Date: {d}")
+                    print(f"Associated Title: {d.content_object.title}")
+                    rid = d.metadata.resource.short_id
+                    print(f"Associated Resource: {SITE_URL}/resource/{rid}")
+                except Exception as e:
+                    print("Error inspecting date: ", e)
+                    pass
+            print("*" * 80)
+        public_count = 0
+        private_count = 0
+        bad_count = 0
+        print(f"Publications between {start_date} and {end_date}")
+        dates = Date.objects.filter(type='created', start_date__range=[start_date, end_date])
```

Comment:
It looks like the bad, public, private logic should be dropped to the if verbose block as it isn't used otherwise.